pwa/src/apiService/resources/applications.ts

```diff
@@ -30,4 +30,5 @@ const deletes = [
   { name: "resultDisplayLayout" },
   { name: "dependenciesDisplayLayout" },
   { name: "landingDisplayLayout" },
+  { name: "catagoryDisplayLayout" },
```

Comment:
This entire array is never used, so I think instead of adding this item to the array, we can just completely remove this array.